pkgs/applications/video/glaxnimate/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, zlib
+, potrace
+, ffmpeg
+, libarchive
+, python3
+, qtbase
+, qttools
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "glaxnimate";
+  version = "0.5.1";
+
+  src = fetchFromGitLab {
+    owner = "mattbas";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "G4ykcOvXXnVIQZUYpRIrALtDSsGqxMvDtcmobjjtlKw=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+  buildInputs = [
+    zlib
+    potrace
+    ffmpeg
+    # gst_all_1.gst-libav # Note that since gst-libav-1.6, libav is actually ffmpeg
+    libarchive
+    python3
```

Comment:
it is a binary so nativeBuildInputs ? same for qttools ?

Reply:
I'd imagine that `python3` is needed at runtime for scripting.

Reply:
Yeah, I agree with @turion , these are libraries/binaries used at runtime so I see no reasons why to put them in `nativeBuildInputs` that should only contain stuff needed to configure the build process (see some files provided by qttools here for instance https://github.com/qt/qttools/blob/dev/src/qtdiag/qtdiag.cpp)